internal/internal_worker_base.go

```diff
@@ -209,6 +215,12 @@ func (bw *baseWorker) Start() {
 	bw.shutdownWG.Add(1)
 	go bw.runTaskDispatcher()
 
+	// We want the emit function run once per host instead of run once per worker
+	emitOnce.Do(func() {
```

Comment:
how about move emitOnce to `bw.emitHardwareUsage`

Reply:
I have tried that, but that results in go routine leak unfortunately